fern/pages/docs/getting-started/docs-overview.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Fern Docs Overview
+description: Learn about key features of Fern's Docs
+subtitle: A static site generator for developer-focused companies to build and host a beautiful, professional docs site.
+layout: overview
+---
+
+Fern Docs supports multiple content types, including:
```

Comment:
What about something like: "Fern Docs provides versatile documentation to support all your content needs, including popular formats like:"